scheduler/scheduler-examples/src/main/java/org/ow2/proactive/scheduler/examples/FTPConnector.java

```diff
@@ -288,10 +290,19 @@ private void createRemoteDirectoryIfNotExists(FTPClient ftpClient, String dirPat
     }
 
     private void createRemoteDirectory(FTPClient ftpClient, String remoteDirPath) throws IOException {
-        if (ftpClient.makeDirectory(remoteDirPath)) {
-            getOut().println("CREATED the directory: " + remoteDirPath);
+        ArrayList<FTPFile> folderList = new ArrayList(Arrays.asList(ftpClient.listDirectories()));
+        ArrayList<String> folderNames = new ArrayList();
+        for (int i = 0; i < folderList.size(); i++) {
+            folderNames.add(folderList.get(i).getName());
+        }
```

Comment:
Maybe you can do like this:
`List<String> folderNames = folderList.stream().map(folder -> folder.getName()).collect(Collectors.toList())`